gpii/node_modules/packagekit/nodepackagekit/nodepackagekit.cc

```diff
@@ -30,12 +38,10 @@ static gboolean DEBUG = FALSE;
 static gchar TAG[] = "[nodePackagekit]";
 
 static std::basic_string<gchar>
-utf8StringFromValue(v8::Handle<v8::Value> value)
+utf8StringFromValue(Handle<Value> value)
 {
-    v8::Local<v8::String> string = value->ToString();
-    gchar *buffer = g_new(gchar, string->Utf8Length());
-    string->WriteUtf8(buffer);
-    return buffer;
+    Local<String> string = value.As<String>();
+    return (*String::Utf8Value(string));
```

Comment:
@amb26, I think you've walked yourself thought it pretty much :)

You're right. Using the nan utilities would make this clearer.  I'll fix it.